web/src/main/java/org/cbioportal/web/interceptor/PagingValidationInterceptor.java

```diff
@@ -0,0 +1,49 @@
+package org.cbioportal.web.interceptor;
```

Comment:
This was discussed for other interface aspect uses, such as security and permission checking. Another candidate: cacheing.